src/PageView/NearMe/NearMeGyms/NearMeGyms.js

```diff
@@ -1,42 +1,49 @@
-import React, { useState, useEffect } from 'react';
-import {
-  GoogleMap,
-  LoadScript,
-  Marker,
-} from '@react-google-maps/api';
-import './NearMeGyms.css';
-import InfoWindows from './InfoWindows/InfoWindows'
+import React, { useState, useEffect } from "react";
+import { GoogleMap, LoadScript, Marker } from "@react-google-maps/api";
+import "./NearMeGyms.css";
+import InfoWindows from "./InfoWindows/InfoWindows";
 
 export default function NearMeGyms() {
   const [data, setData] = useState([]);
   const [location, setLocation] = useState({ lat: 51.5007, lng: -0.1246 });
-  const [markerId, setMarkerId] = useState('')
+  const [markerId, setMarkerId] = useState("");
+  const [query, setQuery] = useState("gym");
+  const [title, setTitle] = useState("Gyms");
+
   useEffect(() => {
     let successPosition = (position) => {
       setLocation({
         lat: position.coords.latitude,
         lng: position.coords.longitude,
       });
-    }
+    };
     let errorHandle = () => setLocation({ lat: 51.5007, lng: -0.1246 });
     navigator.geolocation.getCurrentPosition(successPosition, errorHandle);
   }, []);
 
   useEffect(() => {
     (async () => {
       const response = await fetch(
-        `https://discover.search.hereapi.com/v1/discover?at=${location.lat},${location.lng}&q=gym&limit=15&apiKey=bYn1_nn2_CYDA29oVvcibuL4otpQL3F0HIF5aBnlcW4`
+        `https://discover.search.hereapi.com/v1/discover?at=${location.lat},${location.lng}&q=${query}&limit=15&apiKey=bYn1_nn2_CYDA29oVvcibuL4otpQL3F0HIF5aBnlcW4`
```

Comment:
The URL is very large, please separate it within tokens, e.g.:

`let endpoint = 'http://...`
`let queryParams` = 'limit=15&...'